rules/move_on_aptos/lang/security/signer-leak.yaml

```diff
@@ -40,13 +47,4 @@ rules:
         - pattern: |
             public fun $FUN(...): (..., signer, ...)
         - pattern: |
-            public fun $FUN(...): (..., &signer, ...)
-      #
-      # Metavariable matching reference is broken on the Semgrep side. For now, we
-      # will manually enumerate the patterns (`signer` and `&signer`).
-      #
-      # - metavariable-pattern:
-      #     metavariable: $RETURN
-      #     pattern-either:
-      #       - pattern: signer
-      #       - pattern: "&signer"
```

Comment:
Why this comment has been removed?

Reply:
Clean up. The rule works perfectly right now. No need to use `metavariable-pattern`

Reply:
>The rule works perfectly right now. No need to use metavariable-pattern.
The comment was suggesting using a more concise way of writing the patterns. Either verify that this is fixed and use metavariables, or maintain this comment.